plugins/plugins.go

```diff
@@ -395,6 +396,11 @@ func WithHooks(hs hooks.Hooks) func(*Manager) {
 	}
 }
 
+func (m *Manager) WithParserOptions(opts ast.ParserOptions) *Manager {
```

Comment:
Is there a reason to have this method on the manager instance and not a public constructor. Also a comment would be good.

Reply:
Making it a `func(*Manager)` constructor `opts` arg type, you mean? Yeah, looks like that'd fit better with the established pattern. Will make the change 👍 .